include/swift/AST/DiagnosticsSema.def

```diff
@@ -663,6 +663,9 @@ ERROR(super_initializer_not_in_initializer,none,
       "'super.init' cannot be called outside of an initializer", ())
 WARNING(isa_is_always_true,none, "'%0' test is always true",
         (StringRef))
+WARNING(isa_is_foreign_check,none,
+      "'%0' test always succeeds because %1 is a foreign type",
```

Comment:
Do you want to leave the conditional cast warning in?  On the one hand the cast always succeeds, on the other it is the form we want people to be using.